package.json

```diff
@@ -100,6 +100,7 @@
     "qunit-dom": "^0.6.2",
     "sanitize-html": "^1.14.1",
     "semantic-ui-ember": "^2.1.0",
+    "torii": "^0.10.1",
```

Comment:
We really don't need `torii` for this. We only have to handle the redirect URI. That's it. Which can be done just fine on Ember. Please remove torii dependency

Reply:
Okay :sweat_smile: . Only some minor things are left. I finish this quickly so that this PR can be reviewed as a whole. Thanks.